frontend/app/src/components/home/profile/AccountLinkingCode.svelte

```diff
@@ -0,0 +1,220 @@
+                    </div>
+                    {#if "success" === codeStatus}
+                        <p class="subtitle">
+                            <Translatable resourceKey={i18nKey("accountLinkingCode.webModal.subtitle")} />
+                        </p>
+                    {/if}
+                </div>
+            {/snippet}
+            {#snippet body()}
+                <div class="code-content {!remaining ? 'expired' : ''}">
+                    {#if "success" === codeStatus}
+                        <div class="code-chars">
+                            {#each accountLinkingCode.value as char}
+                                <div class="char">{char}</div>
+                            {/each}
+                        </div>
+                        <div class="remaining">
+                            {#if !remaining}
+                                <Translatable resourceKey={i18nKey("accountLinkingCode.webModal.expired")} />
+                            {:else}
+                                <Translatable resourceKey={i18nKey("accountLinkingCode.webModal.willExpireIn", { remaining: formatRemaining()})} />
+                            {/if}
+                        </div>
+                    {:else if "error" === codeStatus}
+                        <div class="error">
+                            <div class="msg">
+                                <Translatable resourceKey={i18nKey("accountLinkingCode.webModal.error.msg")} />
+                            </div>
+                        </div>
+                    {/if}
+                </div>
+            {/snippet}
+            {#snippet footer()}
+            <div class="footer">
+                {#if "success" === codeStatus}
+                    <Button onClick={modalClose} fill={true} danger={!remaining}>
+                        <Translatable resourceKey={i18nKey("accountLinkingCode.webModal.close")} />
+                    </Button>
+                {:else if "error" === codeStatus}
+                    <Button onClick={fetchAccountLinkingCode} loading={loadingCode}>
+                        <Translatable resourceKey={i18nKey("accountLinkingCode.webModal.error.tryAgain")} />
+                    </Button>
+                    <Button onClick={modalClose} secondary={true}>
+                        <Translatable resourceKey={i18nKey("accountLinkingCode.webModal.error.cancel")} />
+                    </Button>
+                {/if}
+            </div>
+            {/snippet}
+        </ModalContent>
+    </Overlay>
+{/if}
+
+<div class="account-linking">
+    <p class="para">
+        <Translatable resourceKey={i18nKey("accountLinkingCode.settingsMenu.disclaimer")} />
+    </p>
+    <Button onClick={showModal} loading={"initialLoad" === modalStatus}>
+        <Translatable resourceKey={i18nKey("accountLinkingCode.settingsMenu.cta")} />
+    </Button>
+</div>
+
+<style lang="scss">
+    :global(.modal-content) {
```

Comment:
Don't think that we can do this because it will affect _all_ instances of `.modal-content` - you will need to scope it to avoid that.